bin/metamath-compress

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# Modified from https://github.com/metamath/set.mm/blob/develop/scripts/verify
+
+set -euo pipefail
+
+# We should find a cleaner way of doing this long term.
```

Comment:
How about we merge this with how I implemented it in `qed.sh`? My version looks more looks more compact.